pkg/minibroker/minibroker.go

```diff
@@ -108,20 +127,6 @@ func NewClient(
 	}
 }
 
-func loadInClusterClient() kubernetes.Interface {
```

Comment:
FWIW, we could have passed both clients back instead. (Not an issue to inline it, of course.)